SwiftCompilerSources/Sources/Optimizer/Utilities/OptUtils.swift

```diff
@@ -809,3 +809,12 @@ extension CheckedCastAddrBranchInst {
     }
   }
 }
+
+extension Type {
+  func shouldExpand(_ context: some Context) -> Bool {
```

Comment:
Can you add a comment what `shouldExpand` actually means? E.g. copy the comment from the C++ implementation